config/environments/development.rb

```diff
@@ -56,5 +56,6 @@
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
-  Rails.application.routes.default_url_options[:host] = ENV['HOST'] || 'localhost:3002'
+  config.x.application.host = ENV['HOST'] || 'localhost:3002'
```

Comment:
I may be wrong here:
There is a `PORT` environment variable too. Should the development host be `'localhost'` with no port. And then `Rails.application.routes.default_url_options[:port]` defined separately?